### DISCUSSION TITLE
마이페이지 뷰 스케치

### DISCUSSION THREAD

**Discussion by dannaward** [TeamPophory/pophory-iOS#4] [View Sketch]
### 빠진 거, 의견, 질문, 태클 환영

<br/>
<img width="758" alt="image" src="https://github.com/TeamPophory/pophory-iOS/assets/71601985/84ec99c6-1bb7-43bb-a1be-35138d5290f1">

**Comment by joonBaek12** [comment]
한번에 속성까지..텍스트로 넣어놓으니까 또 깔끔해보이네요 ㅎㅎ 배워갑니당
@dannaward navigationItem.titleView에 스택뷰를 넣는 방식으로 이해했는데 맞나요?!
